.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
It is good that you have the proper settings for work in VSCode, however, you should not push it to your remote repo. 
Pull Request represents potential changes in the files in the main branch, and we do not need your individual setting in the common area. 
Please, delete extra files.

Reply:
I deleted this file